fhir-tools/pom.xml

```
@@ -113,6 +107,19 @@
                                         <execute />
                                     </action>
                                 </pluginExecution>
+                                <pluginExecution>
+                                    <pluginExecutionFilter>
+                                        <groupId>org.apache.maven.plugins</groupId>
+                                        <artifactId>maven-plugin-plugin</artifactId>
+                                        <versionRange>3.6.0</versionRange>
```

```suggestion
                                        <versionRange>[3.6.0,)</versionRange>
```
let's match the pattern in checkstyle.